app/util.py

```diff
@@ -56,31 +65,36 @@ def delete_db():
 def read_file(file_name):
     """Given the path of a file, it will be loaded and returned as DataFrame
     Duplicates of emails will be altomatically removed"""
-
-    header = ['firstname','lastname','email']
+    with open(file_name, "r") as f:
```

Comment:
I think we can use pandas to read the csv (pd.read_csv) and get the column names from there, if no key column(s) is specified by the user, we will use the index (which will be just 0,1,2,3,4... by default) of the pandas DataFrame as a key. We can name it `default_id`.

Reply:
I was worried about this, but in my testing, I found that pandas is smart enough to figure out that the nameless column is just an index on its own.